emcc.py

```diff
@@ -862,7 +862,7 @@ def optimizing(opts):
           newargs[i] = newargs[i + 1] = ''
           if key == 'WASM_BACKEND=1':
             exit_with_error('do not set -s WASM_BACKEND, instead set EMCC_WASM_BACKEND=1 in the environment')
-    newargs = [arg for arg in newargs if arg is not '']
+    newargs = [arg for arg in newargs if arg != '']
```

Comment:
hmm.. I wonder why python2 flake8 didn't catch this one?

Reply:
The version is too old.